CONTRIBUTING.md

```diff
@@ -4,16 +4,25 @@
 
 Please submit new issues on the GitHub issue tracker: https://github.com/puppetlabs/bolt/issues
 
-Internally, Puppet uses JIRA for tracking work, so nontrivial bugs or enhancement requests may migrate to JIRA tickets 
-in the "BOLT" project: https://tickets.puppetlabs.com/browse/BOLT/ 
+Internally, Puppet uses JIRA for tracking work, so nontrivial bugs or enhancement requests may migrate to JIRA tickets
+in the "BOLT" project: https://tickets.puppetlabs.com/browse/BOLT/
 
 ## Pull Requests
 
 Pull requests are also welcome on GitHub: https://github.com/puppetlabs/bolt
 
-As with other open-source projects managed by Puppet, Inc we require contributors to digitally sign the Contributor 
+As with other open-source projects managed by Puppet, Inc we require contributors to digitally sign the Contributor
 License Agreement before we can accept your pull request: https://cla.puppet.com
 
+## Pulling down the code
```

Comment:
I believe we already have this covered in the README and installation docs per the last update in https://github.com/puppetlabs/bolt/pull/63